cmake/Modules/FindTinyXML.cmake

```diff
@@ -57,6 +57,9 @@ if (NOT TinyXML_FOUND)
   if(TinyXML_ROOT_DIR)
     set(TinyXML_INCLUDE_PATH PATHS "${TinyXML_ROOT_DIR}/include" NO_DEFAULT_PATH)
     set(TinyXML_LIBRARY_PATH PATHS "${TinyXML_ROOT_DIR}/lib"     NO_DEFAULT_PATH)
+  else()
+    set(TinyXML_INCLUDE_PATH "")
+    set(TinyXML_LIBRARY_PATH "")
```

Comment:
I generally like to default initialize them before the `if` to save the `else` clause but that's just a nitpick.

Reply:
Ops, that is not even the diff I wanted to push. It looks like I already had a branch with the same name an didn't notice when checking out / committing / pushing. I just forced push the branch with the desired change.